docs/root/intro/version_history.rst

```diff
@@ -30,6 +30,9 @@ Version history
 * ext_authz: disabled the use of lowercase string matcher for headers matching in HTTP-based `ext_authz`.
   Can be reverted temporarily by setting runtime feature `envoy.reloadable_features.ext_authz_http_service_enable_case_sensitive_string_matcher` to false.
 * fault: added support for controlling abort faults with :ref:`HTTP header fault configuration <config_http_filters_fault_injection_http_header>` to the HTTP fault filter.
+* grpc-json: added support for building HTTP request into
```

Comment:
?

Reply:
hmm that probably happened during a merge (that changelog entry wasn't in the right place so I moved it around -- fixing now). Thanks for catching!